source/Calamari/Calamari.csproj

```diff
@@ -39,7 +39,7 @@
     <PackageReference Include="OctoDiff" Version="1.1.2" />
     <PackageReference Include="FSharp.Compiler.Tools" Version="4.0.0.1" />
     <PackageReference Include="Octopus.Server.Core.Utilities" Version="1.0.0" />
-    <PackageReference Include="Octopus.Versioning" Version="3.0.2-moveversionrange0004" />
+    <PackageReference Include="Octopus.Versioning" Version="4.0.3-namespace0002" />
```

Comment:
This should bump to 4.0.4, or at the very least be 4.0.3-namespace0003 like in the OctopusDeploy project.

Reply:
Yep. plan is to stabilize all the changes everywhere, then bump up versions across the board accordingly 👍